Example/Example/EditorDemoController.swift

```diff
@@ -181,7 +181,7 @@ class EditorDemoController: UIViewController {
             editorView.richTextView.textColor = UIColor.label
             editorView.richTextView.blockquoteBackgroundColor = UIColor.secondarySystemBackground
             editorView.richTextView.preBackgroundColor = UIColor.secondarySystemBackground
-            editorView.richTextView.blockquoteBorderColor = UIColor.secondarySystemFill
+            editorView.richTextView.blockquoteBorderColors = [.red, .blue, .green]
```

Comment:
To keep the demo consistent with what we have before can we put the `UIColor.secondarySystemFill` as the first color on the array, and then for other colors use the system version to adapt to the dark/light modes.
So it should be something like this in the end:
```
[UIColor.secondarySystemFill, UIColor.systemRed, UIColor.systemBlue, UIColor.systemGreen]
```

Reply:
OK I dropped the red because it contrasts a lot and just use teal and blue which go with the gray better I think for default theme